pandas/_libs/tslibs/offsets.pyx

```diff
@@ -3688,13 +3717,38 @@ cdef class CustomBusinessDay(BusinessDay):
         List/array of dates to exclude from the set of valid business days,
         passed to ``numpy.busdaycalendar``.
     calendar : np.busdaycalendar
-    offset : timedelta, default timedelta(0)
```

Comment:
why remove `offset`? I think it works, e.g.
```python
In [18]: pd.Timestamp(2022, 8, 5, 16) + pd.offsets.CustomBusinessDay(1)
Out[18]: Timestamp('2022-08-08 16:00:00')

In [19]: pd.Timestamp(2022, 8, 5, 16) + pd.offsets.CustomBusinessDay(1, offset=timedelta(days=1))
Out[19]: Timestamp('2022-08-09 16:00:00')
```

Reply:
Thank you for the comment. It was my mistake, I thought offset doesn’t have an effect on `CustomBusinessDay`. I corrected the list of parameters and add an example for offset.

I noticed, that we don't have offset in the list of parameters in `BusinessDay`. Maybe it would be good to add it?

Reply:
yup, sounds good, thanks for checking!